x/emissions/keeper/keeper.go

```diff
@@ -526,6 +533,52 @@ func (k *Keeper) DeleteUnfulfilledReputerNonces(ctx context.Context, topicId Top
 	return k.unfulfilledReputerNonces.Set(ctx, topicId, types.ReputerRequestNonces{})
 }
 
+/// INCLUSIONS
+
+func (k *Keeper) GetCountInfererInclusionsInTopic(ctx context.Context, topicId TopicId, inferer ActorId) (uint64, error) {
+	key := collections.Join(topicId, inferer)
+	count, err := k.countInfererInclusionsInTopic.Get(ctx, key)
+	if err != nil {
+		if errors.Is(err, collections.ErrNotFound) {
+			return 0, nil
+		}
+		return 0, err
+	}
+	return count, nil
+}
+
+func (k *Keeper) IncrementCountInfererInclusionsInTopic(ctx context.Context, topicId TopicId, inferer ActorId) error {
+	key := collections.Join(topicId, inferer)
+	count, err := k.GetCountInfererInclusionsInTopic(ctx, topicId, inferer)
+	if err != nil {
+		return err
+	}
+	count++
+	return k.countInfererInclusionsInTopic.Set(ctx, key, count)
+}
+
+func (k *Keeper) GetCountForecasterInclusionsInTopic(ctx context.Context, topicId TopicId, forecaster ActorId) (uint64, error) {
+	key := collections.Join(topicId, forecaster)
+	count, err := k.countForecasterInclusionsInTopic.Get(ctx, key)
+	if err != nil {
+		if errors.Is(err, collections.ErrNotFound) {
+			return 0, nil
+		}
+		return 0, err
+	}
+	return count, nil
+}
+
+func (k *Keeper) IncrementCountForecasterInclusionsInTopic(ctx context.Context, topicId TopicId, forecaster ActorId) error {
```

Comment:
function header comment describing what this function does

unit test in keeper_test ensuring it operates as expected